runtime/Cpp/demo/TLexer.g4

```diff
@@ -66,7 +66,7 @@ OpenCurly: '{' -> pushMode(Mode1);
 CloseCurly: '}' -> popMode;
 QuestionMark: '?';
 Comma: ',' -> skip;
-Dollar: '$' -> more, mode(Mode1), type(DUMMY);
```

Comment:
I guess the tokens section
```ANTLR
tokens {
	DUMMY	
}
```
can be also removed for now. It's not used anywhere.